torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1242,14 +1540,66 @@ Error PyTorchModelLoader::loadQuantizedLinear(const torch::jit::Node *ptNode) {
                                           {input.dims()[0], weight.dims()[1]},
                                           outScale, outZeroPoint - OFFSETSHIFT);
 
+  // TODO Rowwise quantization is not enabled and accuracy is not tested
+  // We should enable it later.
+  isRowwiseQuantized = false;
```

Comment:
Since this isn't getting tested or used at all, I don't really see the point in putting this in this pr

Reply:
ok I can remove it ;)